src/views/pages/esm.html

```diff
@@ -69,7 +69,7 @@ <h3 class="subtitle">
 							<h4 class="feature-title">Global Multi-CDN powered by jsDelivr</h4>
 
 							<div class="feature-content">
-								esm.run is powered by jsDelivr and it's sponsors like Cloudflare, Fastly, StackPath, Quantil, and appfleet.
+								esm.run is powered by jsDelivr and it's sponsors like bunnynet, Cloudflare, Fastly, Quantil, and appfleet.
```

Comment:
wrong order 🙄

Reply:
The order seems to be alphabetical, is it just coincidence and the order should be Cloudflare, Fastly, Bunnynet, Quantil and appfleet everywhere?

Reply:
> add Bunny (after Cloudflare, Fastly)

it isn't really AB